src/schema.ts

```diff
@@ -9,19 +9,22 @@ import {
   TokensPayload,
 } from "./core/dto/auth.dto.js";
 import {
+  CreateCommentInput,
   CreatePostInput,
   UpdatePostInput,
 } from "./core/dto/social-feed.dto.js";
 import { CreateUserInput } from "./core/dto/user.dto.js";
+import { Comment } from "./core/entities/comment.entity.js";
 import { Post } from "./core/entities/post.entity.js";
 import { User } from "./core/entities/user.entity.js";
 import { AuthResolver } from "./modules/auth/auth.resolver.js";
 import { ProfileResolver } from "./modules/profile/profile.resolver.js";
+import { CommentResolver } from "./modules/social-feed/comment/comment.resolver.js";
 import { UserResolver } from "./modules/user/user.resolver.js";
 
 // the types and resolvers specified here are what will be auto-generated in the graphql schema file
 export const schema = await buildSchema({
-  resolvers: [AuthResolver, ProfileResolver, UserResolver],
+  resolvers: [AuthResolver, ProfileResolver, UserResolver, CommentResolver],
```

Comment:
Oksana had forgotten to include PostResolver here, so if you would like to work with it until she pushes that code change, you can manually add it in here.